.gitignore

```diff
@@ -145,4 +145,6 @@ cython_debug/
 misc/DeployMe/
 
 # JetBrains Ide
-.idea/
\ No newline at end of file
+.idea/
+
+package-lock.json
```

Comment:
`package-lock.json` is generated when you use `npm`. You can remove the file because we are using `yarn`.